### ISSUE TITLE
App api is not working (trying to get full details of application)

### ISSUE THREAD

**Issue by dsharkov** [facundoolano/google-play-scraper#291]
I think Google Play Market changed template, so an error have been occurred.

* Operating System: Windows 10
* Node version: 8.11.2
* google-play-scraper version: 6.2.2

### Description: 
I always get an exception on **app** api.

### Example code:
```js
var gplay = require('google-play-scraper');

gplay.app({appId: 'com.dxco.pandavszombies'})
  .then(console.log, console.log);
```

### Error message:

```
ex  TypeError: Cannot read property 'replace' of undefined
    at Object.descriptionText [as fun] (D:\TestApi\node_modules\google-play-scraper\lib\app.js:181:51)
    at R.map (D:\TestApi\node_modules\google-play-scraper\lib\app.js:175:17)
    at XWrap.f (D:\TestApi\node_modules\ramda\dist\ramda.js:5955:28)
    at XWrap.@@transducer/step (D:\TestApi\node_modules\ramda\dist\ramda.js:392:25)
    at _arrayReduce (D:\TestApi\node_modules\ramda\dist\ramda.js:5137:46)
    at _reduce (D:\TestApi\node_modules\ramda\dist\ramda.js:5167:24)
    at map (D:\TestApi\node_modules\ramda\dist\ramda.js:5954:20)
    at D:\TestApi\node_modules\ramda\dist\ramda.js:586:23
    at Object.f2 [as map] (D:\TestApi\node_modules\ramda\dist\ramda.js:473:22)
    at extractFields (D:\TestApi\node_modules\google-play-scraper\lib\app.js:169:12)
```

Please fix it ASAP because this is very important part of my app.
Thanks in advance.

**Comment by b-reynolds** [comment]
This should be fixed in 6.2.3 

https://github.com/facundoolano/google-play-scraper/commit/b74bc447f7ae5343c3507d6eee30a293c8af7c96

**Comment by yurganov** [comment]
@dsharkov

Why you think somebody must fix your problem ASAP?

And if it is very important part of your project, you should try to dive into it, and try to fix it yourself.

Have a good day.

**Comment by dsharkov** [comment]
> This should be fixed in 6.2.3
> 
> #290

Thanks @b-reynolds
